lib/utils/apiUtils.js

```diff
@@ -66,7 +66,7 @@ async function refreshTokens(instance, firmId) {
       access_token: firmTokens.accessToken,
     };
     const response = await instance.post(
-      `https://api.getsilverfin.com/f/${firmId}/oauth/token`,
+      `https://${BASE_URL}/f/${firmId}/oauth/token`,
```

Comment:
Should the `https://` at the start be removed as already assigned `https://live.getsilverfin.com` to `BASE_URL`?